declarations/plugins/AssetModulesPlugin.d.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * This file was automatically generated.
+ * DO NOT MODIFY BY HAND.
+ * Run `yarn special-lint-fix` to update
+ */
+
+/**
+ * Function that executes for module source abd resource and should return new module source
+ *
+ * This interface was referenced by `AssetModulesPluginOptions`'s JSON-Schema
+ * via the `definition` "DataUrlFn".
+ */
+export type DataUrlFn = (
+	source: string | Buffer,
+	resourcePath: string
+) => string | Buffer | null;
+
+export interface AssetModulesPluginOptions {
+	/**
+	 * The options for data url generator
+	 */
+	dataUrl?: false | DataUrlOptions | DataUrlFn;
+}
+/**
+ * This interface was referenced by `AssetModulesPluginOptions`'s JSON-Schema
+ * via the `definition` "DataUrlOptions".
+ */
+export interface DataUrlOptions {
+	/**
+	 * Is data url encoding enabled (true by default)
+	 */
+	enabled?: boolean;
```

Comment:
Why we need this? If `dataUrl` is not false it is means data-url enbaled

Reply:
yep, I'll drop it